inst/pages/exercises.qmd

```diff
@@ -1703,17 +1706,27 @@ Create an interactive heatmap.
 #| code-summary: "Show solution"
 #| eval: false
 
-# Load necessary libraries
+# Load required libraries
+library(mia)
 library(heatmaply)
 
-# Create example data
-example_data <- matrix(rnorm(200), nrow = 20, ncol = 10)
+# Load the GlobalPatterns dataset from the mia package
+data("GlobalPatterns", package = "mia")
+
+# Here we extract the first 20 taxa and the first 10 samples as an example
+taxa_data <- assay(GlobalPatterns)[1:20, 1:10]
+taxa_names <- rownames(taxa_data)
+sample_names <- colnames(taxa_data)
 
-# Create interactive heatmap
-heatmaply(example_data)
+# Create the interactive heatmap using heatmaply
+heatmaply(taxa_data,
+          xlab = "Samples",
+          ylab = "Taxa",
+          main = "Interactive Heatmap of
+                  GlobalPatterns Dataset")
```

Comment:
main is slightly misleading since this is a subset of GlobalPatterns and the entire GlobalPatterns